backend/app/tests/utils/llm_provider.py

```
@@ -0,0 +1,198 @@
+
+def generate_openai_id(prefix: str, length: int = 40) -> str:
+    """Generate a realistic ID similar to OpenAI's format (alphanumeric only)"""
+    # Generate random alphanumeric string
+    chars = string.ascii_lowercase + string.digits
+    random_part = "".join(secrets.choice(chars) for _ in range(length))
+    return f"{prefix}{random_part}"
+
+
+def mock_openai_assistant(
+    assistant_id: str = "assistant_mock",
+    vector_store_ids: Optional[list[str]] = ["vs_1", "vs_2"],
```

Ruff (0.14.13)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)